### ISSUE TITLE
Installed executable cannot find libchronolog_client.so

### ISSUE THREAD

**Issue by fkengun** [grc-iit/chronolog#232]
### Summary

`LD_LIBRARY_PATH` has to be manually set before running any ChronoLog binaries from the installation directory.

### Steps to reproduce
Running `${INSTAL_PATH}/bin/client_admin` fails with an error message like this
```shell
/home/kfeng/chronolog/Debug/bin/client_admin: error while loading shared libraries: libchronolog_client.so: cannot open shared object file: No such file or directory
```

### What is the current *bug* behavior?
Binaries have to be run with `LD_LIBRARY_PATH` configured properly like this
```shell
LD_LIBRARY_PATH=~/chronolog/Debug/lib ~/chronolog/Debug/bin/client_admin
```

### What is the expected *correct* behavior?
ChronoLog binaries should be able to run as it is. No additional things need to be set.

### Potential cause

`RPATH` or `RUNPATH` is not configured correctly. Currently there are lines of messages during `make install`. It means the `RPATH` or `RUNPATH` is set to empty which could be the root cause of this issue.
```shell
-- Set runtime path of "/home/kfeng/chronolog/Debug/bin/client_admin" to ""
```

### Potential solution

Set `RPATH` or `RUNPATH` using CMake variable `CMAKE_INSTALL_RPATH` might fix it.

**Comment by ibrodkin** [comment]
is this still an issue?

**Comment by fkengun** [comment]
> is this still an issue?

Yes. We worked around it by setting `LD_LIBRARY_PATH` every time we ran binaries that link to `libchronolog_client.so`. The potential solution may not be correct though.
